pkg/interface/chat/src/js/components/lib/chat-input.js

```diff
@@ -345,16 +360,23 @@ export class ChatInput extends Component {
             onChange={(e, d, v) => this.messageChange(e, d, v)}
           />
         </div>
+        <div className="ml2"
```

Comment:
These images are too big! And too close. Compare with mock with this:

![image](https://user-images.githubusercontent.com/20846414/80647539-a50d3b00-8a3c-11ea-9576-23399921c164.png)

![image](https://user-images.githubusercontent.com/20846414/80647557-ab031c00-8a3c-11ea-9a05-c94cb7eb4341.png)

- The two images should be 16x16, not 24x24. 
- The codeEval image needs to be padded.
- The image here doesn't have the multi-shade bg, it probably wasn't included in the export. Also ensure the export is 4x.

You'll also want to ensure the two buttons are consistent; probably they should both be white:

![image](https://user-images.githubusercontent.com/20846414/80647653-d4bc4300-8a3c-11ea-9d95-02be44378b23.png)

Reply:
@urcades Can I get an image asset with squared off borders?

Reply:
Hopefully this can be accounted for in this issue/feature as a small patch to another feature, but the latest iteration of these that Gav refined actually retain a small radii on 'em:

![icn-img](https://user-images.githubusercontent.com/1195363/80651063-2b2c8000-8a43-11ea-9fe2-83ee884055a3.png)

Additionally, our current icon for "code" is incorrect in that it states "centis" is a rune, when it most definitely is not. If we can replace this with a more standardized `>` symbol, that'd be wonderful:

![icn-code](https://user-images.githubusercontent.com/1195363/80651065-2b2c8000-8a43-11ea-85a0-bb14dded96e5.png)

If this additional replacement is out of scope for this issue, I'm happy to add this in and PR myself!

Reply:
So actually, this might be best expressed that rather than replace the "send image" button with a squared off version, we should replace the actually-incorrect "send code" button to normalize it with this work currently being done.

Reply:
> our current icon for "code" is incorrect in that it states "centis" is a rune, when it most definitely is not

It most definitely is! You might be remembering my feedback about a previous iteration (maybe of the dojo tile?) that used `=%`, which is indeed not a rune.

(sad to see `%=` go...)

Reply:
responded to design comments